cycode/cli/code_scanner.py

```diff
@@ -810,6 +811,7 @@ def get_git_repository_tree_file_entries(
 
 def get_default_scan_parameters(context: click.Context) -> dict:
     return {
+        SCA_DISABLE_RESTORE_DEPENDENCIES_FLAG: context.obj.get(SCA_DISABLE_RESTORE_DEPENDENCIES_FLAG),
         'monitor': context.obj.get('monitor'),
```

Comment:
i think we should do it in the same style as is was. without the constant. could be refactored later for **all** options

Reply:
it's cause our product hasn't yet decided on the flag name - it's much easier to change it with constant... I think it's better for most cases